osu.Game.Rulesets.Mania/Edit/Setup/ManiaDifficultySection.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Game.Resources.Localisation.Web;
+using osu.Game.Screens.Edit.Setup;
+
+namespace osu.Game.Rulesets.Mania.Edit.Setup
+{
+    public partial class ManiaDifficultySection : DifficultySection
+    {
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            CircleSizeSlider.Label = BeatmapsetsStrings.ShowStatsCsMania;
```

Comment:
This is fine and all, but if we're doing this, we should probably also hide the AR slider in mania, and both the CS and AR sliders in taiko, as they do absolutely nothing. And rename base velocity in taiko to scroll speed.

Reply:
Yeah, I have a feeling there would be others, which is why I made the implementation allow full control. I'll fix the precision here, but the others should probably be added in a separate pull?

Reply:
Sure, we can do the rest separately.